File: Spaces.DAL/Entities/Comment.cs
using System;
using System.Collections.Generic;

namespace Spaces.DAL.Entities;

public class Comment
{
    public int Id { get; set; }
    public string Title { get; set; }
    public string Content { get; set; }
    public DateTime DateCreated { get; set; }
    public DateTime DateUpdated { get; set; }
    public int PostId { get; set; }
    public Post Post { get; set; }
    public int UserId { get; set; }
    public AppUser User { get; set; }
    public ICollection<LikeForComment> LikedByUsers { get; set; }
}

File: spaces-spa/src/app/_models/Comment.ts
import { LikesForComment } from "./LikesForComment";

export interface Comment {
  id: number;
  title: string;
  content: string;
  createdBy: string;
  dateCreated: Date;
  postId: number;
  userId: number;
  likedByUsers: LikesForComment[];
}
